capstone-frontend/src/components/CreateList.jsx

```diff
@@ -185,6 +185,7 @@ class CreateList extends Component {
                       size="lg"
                       role="status"
                       aria-hidden="true"
+                      variant='primary'
                       className="my-5"
```

Comment:
nit: Since you added this as single quotes, you should change the rest of the props for this spinner to be single quotes - or just make it double quotes since the rest of the file is double quotes and we can change it later

Reply:
Ok